package.json

```diff
@@ -22,6 +22,7 @@
     "@next/bundle-analyzer": "^12.1.0",
     "autoprefixer": "^10.4.0",
     "fuse.js": "^6.6.2",
+    "https": "^1.0.0",
```

Comment:
Ini accidental addition kah?

Reply:
> Ini accidental addition kah?

ngga accidental, kebetulan di sistem gua certificatenya API kominfo ngga valid (kayaknya ngga ada di CA), makanya akhirnya gua nambahin https module buat ngeignore, not the best solution sih tapi